src/components/admin-table/column-def.tsx

```diff
@@ -50,7 +50,19 @@ export const columns: ColumnDef<Reimbursement>[] = [
     header: "Expense Date",
     cell: ({ row }) => (
       <div className="capitalize">
-        {(row.getValue("transactionDate") as Date).toLocaleDateString()}
+        {(() => {
+          const transactionDate = row.getValue("transactionDate");
+          if (
+            typeof transactionDate === "string" ||
+            typeof transactionDate === "number"
+          ) {
+            const date = new Date(transactionDate);
+            return date.toLocaleDateString();
+          }
+          return transactionDate instanceof Date
+            ? transactionDate.toLocaleDateString()
+            : "";
+        })()}
```

Comment:
i don't think we need these conditional checks since the `Date()` constructor can basically turn anything resembling a date into a `Date` object. so we can just do
```
const date = new Date(row.getValue("transactionDate"));
return date.toLocaleDateString();
```

Reply:
Thanks for the tip. Addressed those issues and pushed to this PR